src/tiledimage.js

```diff
@@ -907,13 +924,13 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         this.flipped = flip;
     },
 
-    get flipped(){
+    get flipped() {
         return this._flipped;
     },
-    set flipped(flipped){
+    set flipped(flipped) {
         let changed = this._flipped !== !!flipped;
         this._flipped = !!flipped;
-        if(changed){
+        if (changed && this._initialized) {
```

Comment:
You might think it is not called, but it might be. Flipped is a property, and the constructor calls `$.extend(...)`. If there is a flipped prop in the extension, it invokes this setter.

Reply:
It actually happened to me so that's why it's here.

Reply:
Ok, makes sense